test/ngAnimate/animateCssSpec.js

```diff
@@ -1203,6 +1203,47 @@ describe("ngAnimate $animateCss", function() {
       });
     });
 
+    it('should avoid applying the same cache to an element a follow-up animation is run on the same element',
+      inject(function($animateCss, $rootElement, $document) {
+
+      var element = jqLite('<div></div>');
+      element.attr('class', '');
```

Comment:
No need for this line. You can just do:
```js
var element = jqLite('<div class=""></div>');
```

But what I don't understand is why the test passes without the fix when this class attribute is missing...